cdc/scheduler/internal/v3/scheduler/scheduler_manager.go

```diff
@@ -91,12 +91,12 @@ func (sm *Manager) Schedule(
 			log.Info("schedulerv3: new schedule task",
 				zap.String("namespace", sm.changefeedID.Namespace),
 				zap.String("changefeed", sm.changefeedID.ID),
-				zap.Int("task", len(tasks)),
+				zap.Int("taskNumber", len(tasks)),
+				zap.Any("task", tasks),
```

Comment:
```suggestion
				zap.Int("task", len(tasks)),
```
It might be a very long log.